Sources/Basics/Sandbox.swift

```diff
@@ -133,7 +133,8 @@ fileprivate func macOSSandboxProfile(
     // This is needed to use the UniformTypeIdentifiers API.
     contents += "(allow mach-lookup (global-name \"com.apple.lsd.mapdb\"))\n"
 
-    if allowNetworkConnections.filter({ $0 != .none }).isEmpty == false {
+    //if allowNetworkConnections.filter({ $0 != .none }).isEmpty == false {
```

Comment:
left the original one to make sure I did not get it wrong. will remove when ready to merge